src/continuous_agg.c

```diff
@@ -213,6 +214,203 @@ continuous_agg_init(ContinuousAgg *cagg, const Form_continuous_agg fd)
 	Assert(OidIsValid(cagg->partition_type));
 }
 
+TSDLLEXPORT CaggsInfo
+ts_continuous_agg_get_all_caggs_info(int32 raw_hypertable_id)
```

Comment:
The comments are missing. I suggest adding for every new function (C, SQL) the following:
- The short description of the function
- The description of all arguments
- The description of the return value
- The description of side-effects, if there are any

This is an extra work, but we will benefit a lot from it in the long term.

Reply:
Yes, I plan to add those comments. I was asked to make it ready for review to get some early feedback and because it's a feature branch.